src/components/Swap/SwapActionButton/AllowanceButton.tsx

```diff
@@ -72,7 +68,7 @@ export default function AllowanceButton({ token, isApprovalLoading, approveAndPe
   }, [isApprovalLoading, isFailed, isPending, onClick, pendingApproval, token?.symbol])
 
   return (
-    <ActionButton color={color} disabled={!action?.onClick} action={action}>
+    <ActionButton color={useButtonColor()} disabled={!action?.onClick} action={action}>
```

Comment:
Is ActionButton ever used with a color that is _not_ `useButtonColor`? Or can it be updated to have the color be internal to the component?

Reply:
yes, we have a couple callsites that don't use `useButtonColor`